src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -150,9 +150,11 @@ private ExtendedSocketOptions() {}
 
 
 final class TDS {
-    // TDS protocol versions
-    static final String VER_TDS80 = "tds/8.0"; // TLS-first connections
+    // application protocol
+    static final String PROTOCOL_TDS80 = "tds/8.0"; // TLS-first connections
 
+    // TDS versions
+    static final int VER_TDS80 = 0x8000005; // TDS 8.0
```

Comment:
All the others use the SQL Server codename, should TDS8 follow suit (and use `Dallas`)?

Reply:
Hmm actually, this might be fine since there's only one version, not multiple like 7.2, 7.4. If an 8.x comes out we can change it, otherwise it should be fine.

Reply:
I see, thanks for the explanation!